.travis.yml

```diff
@@ -60,6 +60,9 @@ before_script:
     - git config --global user.name "A U Thor"
     - git config --global user.email "author@example.com"
 
+    # TEMP: replace by sphinx-astropy in PIP_DEPENDENCIES once installed
```

Comment:
once installed or once released?

Reply:
Yep, good catch! I'll fix it before we merge.